cli/script/command-executor.ts

```diff
@@ -551,19 +597,24 @@ function printList<T extends { id: string; name: string; }>(format: string, item
     }
 }
 
-function printAccessKeys<T extends { id: string; name: string; description: string }>(format: string, items: T[]): void {
+function printAccessKeys(format: string, keys: AccessKey[]): void {
     if (format === "json") {
         var dataSource: any[] = [];
 
-        items.forEach((item: T): void => {
-            dataSource.push({ "name": item.name, "id": item.id, "description": item.description });
+        keys.forEach((key: AccessKey): void => {
+            dataSource.push(key);
         });
 
         log(JSON.stringify(dataSource));
     } else if (format === "table") {
-        printTable(["Key", "Description"], (dataSource: any[]): void => {
-            items.forEach((item: T): void => {
-                dataSource.push([item.name, item.description]);
+        printTable(["Key", "Time Created", "Created From", "Description"], (dataSource: any[]): void => {
+            keys.forEach((key: AccessKey): void => {
+                dataSource.push([
+                    key.name, 
+                    key.createdTime ? new Date(+key.createdTime).toString() : "",
```

Comment:
The server was returning a string for the date number representation. This is fixed and no longer needed with the latest server change. :)